spec/models/donation_spec.rb

```diff
@@ -7,4 +7,22 @@
   it { is_expected.to validate_presence_of(:donor) }
   it { is_expected.to validate_presence_of(:amount) }
   it { is_expected.to validate_numericality_of(:amount).is_greater_than(0) }
+
+  describe ".search" do
```

Comment:
Awesome! I would personally group all the donors at the top, and the donations underneath, to aid readability.